examples/MetricsExample/Startup.cs

```diff
@@ -0,0 +1,27 @@
+using MetricsExample.DifferentNamespace;
+using MetricsExample.Model;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Motor.Extensions.Diagnostics.Queue.Metrics;
+using Motor.Extensions.Hosting.Abstractions;
+using Motor.Extensions.Utilities.Abstractions;
+
+namespace MetricsExample
+{
+    public class Startup : IMotorStartup
+    {
+        public void ConfigureServices(WebHostBuilderContext context, IServiceCollection services)
+        {
+            // Add a handler for the input message which returns an output message
+            // This handler is called for every new incoming message
+            services.AddTransient<INoOutputService<InputMessage>, ServiceWithMetrics>();
```

Comment:
I think we should tests, if works move these stuff into the startup class. I had once issue that, i wasn't able to access all services.

Reply:
double checked it and still works as intended I guess - means it works properly :smile: